### DISCUSSION TITLE
Connecting with others<title>

### DISCUSSION THREAD

**Discussion by Kittysbelle** [community/community#46338] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there's a coder/ game maker on here I'd like to message/collaborate with, how do I do that?

**Comment by paule32** [comment]
Hello,
You search for a Game maker ? or do you search for a Person, which can programming Client/Server Application's ?
I have programmed Client/Server Application for years ago, but I am new in Game programming.
So the Information's what is to do, when I would do this are out of date.
And since MS-DOS is out of date, too. The Programming of Games with directly Hardware Access could be difficult under Windows.
But if you like, we can code a own Operating System Simulation, which boots a Game when You insert a Disk of CD into the disk drive,
and boot/start the Computer.
If You like, I give You assistance to collect all neccassary Tools.

I hope I have understand You correctly ...
Happy Codeing, paule32

**Comment by RONIE70** [comment]
Hola!! 
Cómo colaborar en un proyecto en GitHub
Fork del repositorio
Clonar el repositorio
Actualizar la rama master
Crear una rama
Hacer los cambios
Hacer un Pull Request
Fork del repositorio
El primer paso es hacer "Fork" del repositorio.

Clonar el repositorio
Después de tener el repositorio en nuestra cuenta, seleccionar la dirección del repositorio "SSH o HTTP" y clonar:

$ git clone https://github.com/User/NombreRepo.git

Dentro de la carpeta que genera, comprobar la URL del repositorio:

$ git remote -v

Antes de realizar modificaciones agregar la URL del repositorio original del proyecto:

$ git remote add upstream https://github.com/User/RepoOriginal(Forkeado)

Comprobar

$ git remote -v

Actualizar la rama Master
Antes de empezar a trabajar, obtener los últimos cambios del Repo Original:

$ git pull -r upstream master

Crear una Rama
Para crear una rama usar la opción "checkout" de git:

$ git checkout -b feature-nombre-rama

Hacer cambios
Realizar todos los cambios que se desea hacer al proyecto.

Agregar los archivos y hacer un commit

Después de realizar el commit hacer el push hacia nuestro repositorio indicando la rama que hemos creado.

$ git push origin feature-nombre-rama

Hacer un Pull Request
Hacer click en "Compare & Pull Request"

Escribir cambios del Pull Request.

Si todo está bien, enviar con el botón "Send Pull Request".

Esperar a que el duelo del repositorio lo revise, acepte y mezcle en la rama correspondiente.

Espero te sirva. Saludos
